src/CoreWCF.BuildTools/src/OperationParameterInjectionGenerator.Emitter.cs

```diff
@@ -51,12 +53,14 @@ public void Decrement()
             }
 
             private readonly OperationParameterInjectionSourceGenerationContext _sourceGenerationContext;
-            private SourceGenerationSpec _generationSpec;
+            private readonly SourceGenerationSpec _generationSpec;
 
-            public Emitter(in OperationParameterInjectionSourceGenerationContext sourceGenerationContext, SourceGenerationSpec generationSpec)
+            public Emitter(in OperationParameterInjectionSourceGenerationContext sourceGenerationContext, in SourceGenerationSpec generationSpec)
             {
                 _sourceGenerationContext = sourceGenerationContext;
                 _generationSpec = generationSpec;
+                _builder = new StringBuilder();
+                _builder.Clear();
             }
```

Comment:
Why are you calling Clear() on a new StringBuilder instance?

Reply:
Thanks! I moved the `Clear` in the foreach loop.